app/models/product_import/entry_processor.rb

```diff
@@ -135,7 +135,7 @@ def save_new_inventory_item(entry)
         @inventory_created += 1
         @updated_ids.push new_item.id
       else
-        @importer.errors.add("#{I18n.t('admin.product_import.model.line')} #{line_number}:", new_item.errors.full_messages)
+        @importer.errors.add("#{I18n.t('admin.product_import.model.line')} #{entry.line_number}:", new_item.errors.full_messages)
```

Comment:
When duplicating the same code three or more times, I tend to extract it.

I would also use I18n for the string interpolation. Some languages may prefer something like "nth line:" instead of "Line n:".
```diff
-# with en.yml:
-#   line: Line
-"#{I18n.t('admin.product_import.model.line')} #{entry.line_number}:"

+# with en.yml:
+#   line: "Line %{number}:"
+I18n.t('admin.product_import.model.line', number: entry.line_number)
```

Reply:
Yep, good thinking.

Reply:
I realised I've addressed this here and elsewhere in another PR. I'll remove this commit from here and address it (including your I18n suggestion) in https://github.com/openfoodfoundation/openfoodnetwork/pull/2842